README.md

```diff
@@ -27,6 +27,10 @@ By default the plugin only applies this filtering to the `console` task in the `
 scalacOptions.in(Tut) ~= filterConsoleScalacOptions
 ```
 
+Additionally, if you need to temporarily disable fatal warnings.
```

Comment:
It would also be nice to have control over the mode with an environment variable or system property. Thus one can enable relaxed mode for the whole CI or DEV build without modifying sbt command

Reply:
Hey, that is an interesting idea. I will play with it and see if I can implement it.
What would be a good name for the env var? `SBT_TPOLECAT_RELAXED`?

Reply:
`SBT_TPOLECAT_RELAXED` sounds good to me. Thanks for considering!

Reply:
@seveneves Done!

Reply:
I will try it out from a local build. Thanks!

Reply:
works like a charm 😄